### DISCUSSION TITLE
Copy and Paste auto-snapping

### DISCUSSION THREAD

**Discussion by Louis-P-WSU** [ppy/osu#14952] [Ideas]
Just an idea I had, but when you copy and paste objects in the current version of the editor, the time is matched to the ms, but this ends up causing very small snapping errors that later come up in Ai-Mod. Fixing them isn't that big of an issue, I could just tell it to resnap all objects, but this map im working on is an hour long so I don't want it messing up anything, which has happened before.  

Idk if this is a feature in lazer yet or not but the option for a smart paste that auto-snaps your objects when you place the copies so that issue doesn't show up would be nice. 

Perhaps make it a modifier key like, if you've just copied something, when you go to paste, ctrl+shift+v does the smart paste, and then have a smart paste by default toggle.

**Comment by peppy** [comment]
That sounds like something which shouldn't occur in the first place. Nor should it require special handling of on the user's end.
